docs/items/high-resolution-items.md

```diff
@@ -0,0 +1,154 @@
+---
+title: High Resolution Items
+category: Tutorials
+mentions:
+    - BlazeDrake
+---
+
+::: tip
+This tutorial uses attachables. If you do not know what an attachable is, read this page first: [Attachables](attachables.html)
```

Comment:
Changed `attachables.html` to `/items/attachables`

Reply:
Ok, I will implement this change.